### DISCUSSION TITLE
Pagination and infinite scroll

### DISCUSSION THREAD

**Discussion by tanat0** [nocodb/nocodb#10018] [Q&A] [answered]
Since version 0.258.0 pagination was removed. Is it possible to bring pagination back as an option to select? 

Ex. there may be switch between pagination and infinite scroll (in table settings?) or whatever you consider appropriate.

**Comment by o1lab** [comment, marked as answer]
From bottom left - use feature preview. This will give option to toggle between this feature. But its not per table.

<img width="256" alt="image" src="https://github.com/user-attachments/assets/3e8b78bb-3a2f-423e-a4c6-53c2d16b584d">

**Reply by tanat0** [reply]
Oh, my bad
Thank you
